### DISCUSSION TITLE
Access to LPW through other networks

### DISCUSSION THREAD

**Discussion by kevinchu12** [kspviswa/local-packet-whisperer#9] [Q&A] [answered]
I have a question here, at present, LPW is set up with localhost by default, but there is a server in my room which is used to run the open source model by default, and other computers are used to access it, is there any way to let LPW access it through other networks?

Thanks!

**Comment by kspviswa** [comment, marked as answer]
Hey thanks for this question. I have now enabled LPW to connect with ollama server over a network. Please upgrade to latest LPW and you should be seeing a setting to put a remote IP. For more details refer https://github.com/kspviswa/local-packet-whisperer/releases/tag/v0.2.3 .
